src/views/preview/presentation.jsx

```diff
@@ -124,20 +124,22 @@ const PreviewPresentation = ({
                                     }
                                 </div>
                             </FlexRow>
-                            <div className="project-buttons">
-                                {/* TODO: Hide Remix button for now until implemented */}
-                                {(!userOwnsProject && false) &&
-                                    <Button className="button remix-button">
-                                        Remix
+                            <MediaQuery minWidth={frameless.mobile}>
```

Comment:
It seems inconsistent that in `lib/frameless` we define desktop, tablet and mobile. Then in preview css we define screen sizes: small, medium, big (along with medium-and-small). It's a bit of technical debt that I think we should address at some point.
/cc @rschamp 

Also - what does mobile really mean? lots of tablets are also 'mobile' in the sense of having a cellular connection. And there are tablets that are big enough to be 'desktop'.